homeassistant/components/keyboard_remote/__init__.py

```diff
@@ -294,6 +301,7 @@ async def async_monitor_input(self, dev):
                                 KEYBOARD_REMOTE_COMMAND_RECEIVED,
                                 {
                                     KEY_CODE: event.code,
+                                    TYPE: KEY_VALUE_NAME.get(event.value),
```

Comment:
Do we ever expect to get an unknown value here, and if we do should we fall back to some default?

Reply:
This should never happen.  One of the conditions in the loop checks that `event.value` is in the allowed types.

ex: `if event.value in self.key_values:`

Reply:
Then we should not use dict.get(), use [] instead.

Reply:
Ok, updated.